app/uk/gov/hmrc/fileupload/quarantine/events.scala

```diff
@@ -20,7 +20,7 @@ import play.api.libs.json.{Format, JsObject, Json}
 import uk.gov.hmrc.fileupload.{EnvelopeId, Event, FileId, FileRefId}
 
 case class FileInQuarantineStored(envelopeId: EnvelopeId, fileId: FileId, fileRefId: FileRefId,
-                                  created: Long, name: String, contentType: String, metadata: JsObject) extends Event
+                                  created: Long, name: String, fileLength: Long, contentType: String, metadata: JsObject) extends Event
```

Comment:
OK that makes sense (even though that addition should have been made in a different branch)